packages/components/rating/src/Rating.doc.mdx

```diff
@@ -30,4 +31,36 @@ import { Rating } from "@spark-ui/rating"
 ## Usage
 
 ### Default
+
+Rating control lets user see and/or set a star rating on a scale from 0 to 5.
+
 <Canvas of={stories.Default} />
+
+### Readonly
+
+Use `readOnly` prop to indicate that rating control is not interactive. In this case only (non interactive) the rating control will handle 0.5 step (half stars).
+
+<Canvas of={stories.Readonly} />
+
+### Disabled
+
+Use `disabled` prop to indicate that rating control is disabled.
+
+<Canvas of={stories.Disabled} />
+
+### Sizes
+
+Use `size` prop to set the size of the rating control stars.
+
+<Canvas of={stories.Size} />
+
+## Accessibility
+
+Follows WCAG 2.0 standards for keyboard accessibility guidelines.
```

Comment:
I can't find the WCAG link for the keyboard navigation standard of a "Rating", maybe we can add a link to this one instead ? https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/

Reply:
After some researches, we noticed there were two patterns when implementing a rating component: using a slider input, or a group of radio inputs. With @acd02 we chose the first option, that seemed more appropriate to us (and following the Adobe Spectrum implementation). The pattern you linked only fit with second choice... :sweat_smile: